let Validator,HttpError;_6fc‍.x([["default",()=>validate]]);_6fc‍.w("validatorjs",[["default",["Validator"],function(v){Validator=v}]]);_6fc‍.w("http-errors",[["default",["HttpError"],function(v){HttpError=v}]]);


               function validate(input, rules, customMessages) {
  const data = new Validator(input, rules, customMessages);
  if (data.fails()) {
    throw HttpError(422, data.errors);
  }
}
